DotNetCoreMasters/DotNetCoreMasters/Startup.cs

```diff
@@ -19,6 +20,8 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddControllers();
             services.AddItemService();
```

Comment:
what is this for?

Reply:
Register DI for service

Reply:
i mean this line services.AddScoped<ItemDTO>();

Reply:
for ItemDTO

Reply:
updated no services.AddScoped();